deployment/permissions/service.like.like.ini

```diff
@@ -26,3 +26,13 @@ permissionItem3.param5 =
 permissionItem3.tags = 
 permissionItem3.permissions = BASE_USER_SESSION_PERMISSION
 
+permissionItem4.service = like_like
+permissionItem4.action = list
+permissionItem4.partnerId = 0
+permissionItem4.param3 =
+permissionItem4.param4 =
+permissionItem4.param5 =
+permissionItem4.tags =
+permissionItem4.permissions = BASE_USER_SESSION_PERMISSION
```

Comment:
@shpiNi  Are you sure that an action allowing you to list info about all the users belonging to the partner should be granted on such a low-level permission?